### ISSUE TITLE
Twitter icon not showing up on footer

### ISSUE THREAD

**Issue by michael-erasmus** [halogenica/beautifulhugo#36]
This is a odd one, and I hope I'm not doing anything wrong, but for some reason I cannot get the author Twitter social icon to show up in the footer.

I tried a bunch of other social icons (github, reddit,  stackoverflow) and they all seem to work, but the twitter link has no icon.

In my `config.toml`:

[Author]
  name = "Michael Erasmus"
  github = "username"
  twitter = "username"
  reddit = "username"
  stackoverflow = "users/XXXXXXX/username"

This is what I'm seeing on my footer though (when I run `hugo server`)

![png](https://www.dropbox.com/s/he4ki6u6a5g3a54/Screenshot%202017-03-22%2022.38.35.png?dl=1)

The `<i>` tag for the twitter link seems to be there, but it's just not showing an icon?

![png](https://www.dropbox.com/s/tmfln8vbjgolkoq/Screenshot%202017-03-22%2022.43.38.png?dl=1)

**Comment by halogenica** [comment]
someone else had almost this same problem specifically with firefox on linux. Their problem was that only every other icon showed up, and it had the same spacing and appearance as the missing icon screenshot you linked. I checked with the same version of firefox on windows and all was well. The person I believe just rebooted their machine and it was fixed. Truly bizarre. What browser/version/OS are you using, and can you view things correctly in another browser?

**Comment by michael-erasmus** [comment]
Oh wow, thanks for the quick response! 

>  The person I believe just rebooted their machine and it was fixed.

Huh, that is weird, indeed that's what happened with me as well. Funny thing is I was actually on Chrome (version 56.0.2924.87 (64-bit)) on Mac OS Sierra. 

But when I tried in Firefox, I could see the icon! And after a reboot all seemed well too.

What's interesting is that it also worked on Chrome in Incognito mode before I rebooted. Sorry for the false alarm!

Awesome theme by the way, thanks so much for making it.
